lib/publiccloud/provider.pm

```diff
@@ -355,6 +355,7 @@ sub terraform_apply {
         $cmd .= "-var 'region=" . $self->region . "' ";
         $cmd .= "-var 'name=" . $name . "' ";
         $cmd .= "-var 'project=" . $args{project} . "' " if $args{project};
+        $cmd .= sprintf(qq(-var 'tags={"openqa_ttl":"%d"}' ), get_var('MAX_JOB_TIME', 7200));
```

Comment:
@cfconrad thank you for letting me know. So if I understand correctly from this line, the `openqa_ttl` tag is actually equal to `MAX_JOB_TIME` variable? I would maybe add some small extra gap between those - like 5 minutes for example.

Reply:
I thought about that as well, but we always boot a VM which already have a gap. But why not adding some more. I will add 5min

Reply:
You're right! That should be enough. You can add or keep it as is ;)